apps/showcase/package.json

```diff
@@ -92,7 +90,8 @@
     "rxjs": "^7.8.1",
     "sass-loader": "^14.0.0",
     "tslib": "^2.6.2",
-    "zone.js": "~0.14.2"
+    "zone.js": "~0.14.2",
+    "@o3r/apis-manager": "0.0.0-placeholder"
```

Comment:
should be alphabetical order ?

Reply:
also, it should be `"workspace:^"`
I'm not sure why it was added (by Nx I assume), maybe because of the imports in the training exercises?